x-pack/plugins/apm/server/routes/alerts/route.ts

```diff
@@ -15,6 +15,13 @@ import { environmentRt, rangeRt } from '../default_api_types';
 import { AggregationType } from '../../../common/rules/apm_rule_types';
 import { getApmEventClient } from '../../lib/helpers/get_apm_event_client';
 
+const searchConfigurationRt = t.type({
+  query: t.type({
+    query: t.union([t.string, t.record(t.string, t.any)]),
+    language: t.string,
+  }),
+});
+
```

Comment:
Wasn't this already specified in `searchConfigurationSchema`? We should be able to specify once and re-use